docs/pages/api-docs/button-unstyled.json

```diff
@@ -7,10 +7,6 @@
       }
     },
     "component": { "type": { "name": "elementType" }, "default": "'button'" },
-    "components": {
```

Comment:
Was this change intentional?

Reply:
But isn't this file the API of the `ButtonUnstyled`? Based on the name I am a bit confused